workers/proto/worker.pb.go

```diff
@@ -3,13 +3,14 @@
 
 package protobuf
 
+import proto "github.com/golang/protobuf/proto"
```

Comment:
This looks very different from my generation output. Which protoc version do you use? I use 3.7.0

Reply:
❯ protoc --version
libprotoc 3.7.1

And whatever the latest is from protoc-gen-go.